FlaskApp/api/app.py

```diff
@@ -3,22 +3,47 @@
 from flask_jwt_extended import JWTManager
 
 from ..config import config
-from .routes import AllRides, GetRide,  JoinRequest, Register, Login, Usersg
+from .routes import Home, AllRides, GetRide,  JoinRequest, Register, Login
+from .db_config import con
+from .db import create_table
+
 
 
 def create_app(config_name):
     app = Flask(__name__)
     app.config.from_object(config[config_name])
     jwt = JWTManager(app)
     api = Api(app)
+
+    #create table queries
+    user_table_queries=  'CREATE TABLE IF NOT EXISTS users (Id SERIAL PRIMARY KEY, username VARCHAR(255),\
+                      password VARCHAR(255), email VARCHAR(255));'
+
+    ride_offer_queries= 'CREATE TABLE IF NOT EXISTS rides(Id SERIAL PRIMARY KEY, current_location VARCHAR(85),\
+                     destination VARCHAR(255), depature_time VARCHAR(255), seats_available VARCHAR(255),\
+                     user_id SERIAL, FOREIGN KEY (user_id) REFERENCES users(Id), cost INTEGER);'
+
+    request_queries =   'CREATE TABLE IF NOT EXISTS request(Id SERIAL PRIMARY KEY, Username VARCHAR(255),\
+                      current_location VARCHAR(255), destination VARCHAR(255) , depature_time VARCHAR(255),\
+                      request_id SERIAL, FOREIGN KEY(request_id) REFERENCES rides (Id));'
+
+
+
+    create_table(con, user_table_queries)
+    create_table(con, ride_offer_queries)
+    create_table(con, request_queries)
```

Comment:
If you can manage, I'd suggest you separate database migration and app factory. I.e. maybe it would be a good idea not to migrate your DB everytime your app is run. My understanding of data persistence is that your data remains even if your server is restarted a billion times, your server doesn't have to reset the DB everytime as it is in the in-memory situation